src/libraries/System.Private.CoreLib/src/System/Runtime/Serialization/SerializationInfo.cs

```diff
@@ -145,12 +145,12 @@ private void ExpandArrays()
 
         public void AddValue(string name, object? value, Type type)
         {
-            if (null == name)
+            if (name is null)
```

Comment:
I thought we already had an analyzer for "yoda conditionals"

Reply:
There is, SA1131, which is not yet enabled. See https://github.com/dotnet/runtime/issues/40561, and a PR that's been open for almost two months if you want to go help push it forward hint hint https://github.com/dotnet/runtime/pull/40652.

Reply:
Haha - the one with 4700 lines - I already reviewed that and I assumed that we merged it...